apps/web/screens/PetitionView/PetitionView.tsx

```diff
@@ -208,39 +186,67 @@ const PetitionView: Screen<PetitionViewProps> = ({ namespace }) => {
                 </T.Row>
               </T.Head>
               <T.Body>
-                {
-                  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-                  // @ts-ignore make web strict
-                  pagePetitions?.map((petition) => {
-                    return (
-                      <T.Row key={petition.id}>
-                        <T.Data text={{ variant: 'medium' }}>
-                          {formatDate(petition.created)}
-                        </T.Data>
-                        <T.Data text={{ variant: 'medium' }}>
-                          {petition.meta.fullName
-                            ? petition.meta.fullName
-                            : n('noName', 'Nafn ekki skráð')}
-                        </T.Data>
-                      </T.Row>
-                    )
-                  })
-                }
+                {loadingEndorsements &&
+                  Array.from({ length: 10 }, (_, i) => (
```

Comment:
```suggestion
                  Array(10).fill().map((_, i) => (
```

Is this less hacky? IDK.
Why doesn't javascript make this easier :laughing: